source/main/index.js

```diff
@@ -88,6 +92,8 @@ const onAppReady = async () => {
   const startTime = new Date().toISOString();
   // first checks for japanese locale, otherwise returns english
   const systemLocale = detectSystemLocale();
+  // first checks for japanese locale, otherwise returns english
```

Comment:
@DeeJayElly I don't think this comment matches your implementation 🤔

Reply:
@nikolaglumac Yep makes sense. it was a copy/paste. I will change it.

Reply:
@nikolaglumac done